### DISCUSSION TITLE
Edit a page created with the WYSIWYG

### DISCUSSION THREAD

**Discussion by iwkse** [requarks/wiki#3154] [Help / Questions]
Hi,
is is possible to modify a page created with the graphical editor to include HTML source code?
I can't find a way to do that

**Comment by mattpr** [comment]
The editor type (html, markdown wysiwyg) is (currently) set/locked at page creation.  Annoying but the current state.

There is an open pull request to allow switching editor types on edit and it is scheduled for 3.x release (see roadmap).

In terms of what an editor supports features-wise...that is up to the individual editor.

I believe markdown editor should allow you to just paste in raw html.  There might be some sanitization settings you can check under the wiki admin for the various editor types.

**Reply by xstable** [reply]
That would be a nice feature. Also a nice feature would be a Version-Number anywhere in the Backend. 
Do you have any suggestion how to see which version I do currently use.

**Reply by mattpr** [reply]
The feature to switch a page to use a different editor type has been released.  It is called "convert" in the english UI and is under the page actions popup menu.  So if you create a page in WYSIWYG you can convert it to be edited in markdown or vice-versa.  However that editor is tied to the page so every user that wants to edit the page with a different editor may be constantly converting the page back and forth.  

Feels pretty clunky design-wise to be honest.  Would be better if there was a common underlying storage formula such that different editor types could be used and converted to/from the underlying format when opening for edit in a specific editor or saving the result of that editor.  But I doubt this will get redesigned.  Most development on wiki.js is done by a single guy...impressive what he has built, but really polishing some of these things would take a lot of development time....more than realistic for a single developer given the number of things on his wishlist/backlog.

> Also a nice feature would be a Version-Number anywhere in the Backend.

If by backend you mean in the deployed wikijs code on the server, the version is in the usual place...

```
root@wiki:/opt/wiki-js/current# jq '.version' package.json
"2.5.201"
```

If you mean the UI...
The current version is displayed in the admin section of the UI

![image](https://user-images.githubusercontent.com/1196252/119135085-4008f680-ba3e-11eb-847b-9c9e31d8502c.png)
